show/main.py

```diff
@@ -1252,17 +1255,20 @@ def table(verbose):
 
 @cli.command()
 @click.argument('process', required=False)
-@click.option('-l', '--lines')
+@click.option('-l', '--lines', type=int)
 @click.option('-f', '--follow', is_flag=True)
 @click.option('--verbose', is_flag=True, help="Enable verbose output")
 def logging(process, lines, follow, verbose):
     """Show system log"""
+    if process and not re.match(r'^[a-zA-Z0-9\s]+$', process):
+        sys.exit('Process contains only number, alphabet, and whitespace.')
```

Comment:
>exit [](http://example.com/codeflow?start=12&length=4)

What is the reason to add this check?

Reply:
Offline discussed, this check is not accurate. Let's revert back, and please use another future PR to focus on this fix.